comps/comps-foreman-fedora21.xml

```diff
@@ -144,17 +144,17 @@
       <packagereq type="default">rubygem-hirb-doc</packagereq>
       <packagereq type="default">rubygem-hirb-unicode-doc</packagereq>
       <packagereq type="default">rubygem-i18n-doc</packagereq>
-      <packagereq type="default">rubygem-jquery_pwstrength_bootstrap_4-doc</packagereq>
       <packagereq type="default">rubygem-jquery-turbolinks-doc</packagereq>
       <packagereq type="default">rubygem-jquery-ui-rails-doc</packagereq>
+      <packagereq type="default">rubygem-jquery_pwstrength_bootstrap_4-doc</packagereq>
```

Comment:
ditto comment about collation settings and locale, I think hyphens versus underscores is also a cz vs. en issue.